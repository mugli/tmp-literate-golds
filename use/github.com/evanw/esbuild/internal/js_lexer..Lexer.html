<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<title>References: github.com/evanw/esbuild/internal/js_lexer.Lexer</title>
<link href="../../../../../css/light-v0.3.2-preview.css" rel="stylesheet">
<script src="../../../../../jvs/golds-v0.3.2-preview.js"></script>
<body onload="onPageLoad()"><div>

<pre><code><span style="font-size:x-large;">type <b><a href="../../../../../pkg/github.com/evanw/esbuild/internal/js_lexer.html">github.com/evanw/esbuild/internal/js_lexer</a>.<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-218">Lexer</a></b></span>

<span class="title">41 uses</span>

	github.com/evanw/esbuild/internal/js_lexer (current package)
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-218">js_lexer.go#L218</a>: type <b>Lexer</b> struct {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-251" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-251">#L251</a>: func NewLexer(log logger.Log, source logger.Source) <b>Lexer</b> {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-252" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-252">#L252</a>: 	lexer := <b>Lexer</b>{
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-262" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-262">#L262</a>: func NewLexerGlobalName(log logger.Log, source logger.Source) <b>Lexer</b> {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-263" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-263">#L263</a>: 	lexer := <b>Lexer</b>{
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-274" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-274">#L274</a>: func NewLexerJSON(log logger.Log, source logger.Source, allowComments bool) <b>Lexer</b> {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-275" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-275">#L275</a>: 	lexer := <b>Lexer</b>{
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-289" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-289">#L289</a>: func (lexer *<b>Lexer</b>) Loc() logger.Loc {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-293" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-293">#L293</a>: func (lexer *<b>Lexer</b>) Range() logger.Range {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-297" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-297">#L297</a>: func (lexer *<b>Lexer</b>) Raw() string {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-301" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-301">#L301</a>: func (lexer *<b>Lexer</b>) RawTemplateContents() string {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-351" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-351">#L351</a>: func (lexer *<b>Lexer</b>) IsIdentifierOrKeyword() bool {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-355" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-355">#L355</a>: func (lexer *<b>Lexer</b>) IsContextualKeyword(text string) bool {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-359" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-359">#L359</a>: func (lexer *<b>Lexer</b>) ExpectContextualKeyword(text string) {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-366" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-366">#L366</a>: func (lexer *<b>Lexer</b>) SyntaxError() {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-385" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-385">#L385</a>: func (lexer *<b>Lexer</b>) ExpectedString(text string) {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-394" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-394">#L394</a>: func (lexer *<b>Lexer</b>) Expected(token T) {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-402" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-402">#L402</a>: func (lexer *<b>Lexer</b>) Unexpected() {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-411" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-411">#L411</a>: func (lexer *<b>Lexer</b>) Expect(token T) {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-418" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-418">#L418</a>: func (lexer *<b>Lexer</b>) ExpectOrInsertSemicolon() {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-428" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-428">#L428</a>: func (lexer *<b>Lexer</b>) ExpectLessThan(isInsideJSXElement bool) {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-457" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-457">#L457</a>: func (lexer *<b>Lexer</b>) ExpectGreaterThan(isInsideJSXElement bool) {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-686" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-686">#L686</a>: func (lexer *<b>Lexer</b>) ExpectJSXElementChild(token T) {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-693" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-693">#L693</a>: func (lexer *<b>Lexer</b>) NextJSXElementChild() {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-768" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-768">#L768</a>: func (lexer *<b>Lexer</b>) ExpectInsideJSXElement(token T) {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-775" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-775">#L775</a>: func (lexer *<b>Lexer</b>) NextInsideJSXElement() {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-957" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-957">#L957</a>: func (lexer *<b>Lexer</b>) Next() {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-1566" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-1566">#L1566</a>: func (lexer *<b>Lexer</b>) scanIdentifierWithEscapes(kind identifierKind) (string, T) {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-1646" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-1646">#L1646</a>: func (lexer *<b>Lexer</b>) parseNumericLiteralOrDot() {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-1942" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-1942">#L1942</a>: func (lexer *<b>Lexer</b>) ScanRegExp() {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-2084" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-2084">#L2084</a>: func (lexer *<b>Lexer</b>) decodeEscapeSequences(start int, text string) []uint16 {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-2337" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-2337">#L2337</a>: func (lexer *<b>Lexer</b>) RescanCloseBraceAsTemplateToken() {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-2350" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-2350">#L2350</a>: func (lexer *<b>Lexer</b>) step() {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-2373" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-2373">#L2373</a>: func (lexer *<b>Lexer</b>) addError(loc logger.Loc, text string) {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-2385" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-2385">#L2385</a>: func (lexer *<b>Lexer</b>) addErrorWithNotes(loc logger.Loc, text string, notes []logger.MsgData) {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-2397" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-2397">#L2397</a>: func (lexer *<b>Lexer</b>) addRangeError(r logger.Range, text string) {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-2477" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-2477">#L2477</a>: func (lexer *<b>Lexer</b>) scanCommentText() {

	<a href="../../../../../pkg/github.com/evanw/esbuild/internal/js_parser.html">github.com/evanw/esbuild/internal/js_parser</a>
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-38">js_parser.go#L38</a>: 	lexer                    js_lexer.<b>Lexer</b>
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-12622" class="path-duplicate">js_parser.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-12622">#L12622</a>: func newParser(log logger.Log, source logger.Source, lexer js_lexer.<b>Lexer</b>, options *Options) *parser {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-12830" class="path-duplicate">js_parser.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-12830">#L12830</a>: 	p := newParser(log, source, js_lexer.<b>Lexer</b>{}, &options)
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/json_parser.go.html#line-14">json_parser.go#L14</a>: 	lexer   js_lexer.<b>Lexer</b>
</code></pre><pre id="footer">
<table><tr><td><img src="../../../../../png/go101-twitter.png"></td>
<td>The pages are generated with <a href="https://go101.org/article/tool-golds.html"><b>Golds</b></a> <i>v0.3.2-preview</i>. (GOOS=darwin GOARCH=amd64)
<b>Golds</b> is a <a href="https://go101.org">Go 101</a> project developed by <a href="https://tapirgames.com">Tapir Liu</a>.
PR and bug reports are welcome and can be submitted to <a href="https://github.com/go101/golds">the issue list</a>.
Please follow <a href="https://twitter.com/go100and1">@Go100and1</a> (reachable from the left QR code) to get the latest news of <b>Golds</b>.</td></tr></table></pre>