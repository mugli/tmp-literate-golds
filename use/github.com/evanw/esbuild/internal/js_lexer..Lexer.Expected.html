<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<title>References: github.com/evanw/esbuild/internal/js_lexer.Lexer.Expected</title>
<link href="../../../../../css/light-v0.3.2-preview.css" rel="stylesheet">
<script src="../../../../../jvs/golds-v0.3.2-preview.js"></script>
<body onload="onPageLoad()"><div>

<pre><code><span style="font-size:x-large;"><b><a href="../../../../../pkg/github.com/evanw/esbuild/internal/js_lexer.html">github.com/evanw/esbuild/internal/js_lexer</a>.<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-218">Lexer</a>.<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-394">Expected</a></b></span><span style="font-size: large;"><i> (method)</i></span>

<span class="title">21 uses</span>

	github.com/evanw/esbuild/internal/js_lexer (current package)
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-394">js_lexer.go#L394</a>: func (lexer *Lexer) <b>Expected</b>(token T) {
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-413" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-413">#L413</a>: 		lexer.<b>Expected</b>(token)
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-450" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-450">#L450</a>: 		lexer.<b>Expected</b>(TLessThan)
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-487" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-487">#L487</a>: 		lexer.<b>Expected</b>(TGreaterThan)
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-688" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-688">#L688</a>: 		lexer.<b>Expected</b>(token)
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-770" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-770">#L770</a>: 		lexer.<b>Expected</b>(token)
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-2339" class="path-duplicate">js_lexer.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_lexer/js_lexer.go.html#line-2339">#L2339</a>: 		lexer.<b>Expected</b>(TCloseBrace)

	<a href="../../../../../pkg/github.com/evanw/esbuild/internal/js_parser.html">github.com/evanw/esbuild/internal/js_parser</a>
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-1739">js_parser.go#L1739</a>: 			p.lexer.<b>Expected</b>(js_lexer.TIdentifier)
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-2455" class="path-duplicate">js_parser.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-2455">#L2455</a>: 	p.lexer.<b>Expected</b>(js_lexer.TEqualsGreaterThan)
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-3350" class="path-duplicate">js_parser.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-3350">#L3350</a>: 					p.lexer.<b>Expected</b>(js_lexer.TIdentifier)
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-3417" class="path-duplicate">js_parser.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-3417">#L3417</a>: 					p.lexer.<b>Expected</b>(js_lexer.TIdentifier)
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-3421" class="path-duplicate">js_parser.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-3421">#L3421</a>: 					p.lexer.<b>Expected</b>(js_lexer.TOpenParen)
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-4150" class="path-duplicate">js_parser.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-4150">#L4150</a>: 			p.lexer.<b>Expected</b>(js_lexer.TGreaterThan)
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-4207" class="path-duplicate">js_parser.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-4207">#L4207</a>: 				p.lexer.<b>Expected</b>(js_lexer.TGreaterThan)
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-4748" class="path-duplicate">js_parser.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-4748">#L4748</a>: 		p.lexer.<b>Expected</b>(js_lexer.TClass)
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-5061" class="path-duplicate">js_parser.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-5061">#L5061</a>: 			p.lexer.<b>Expected</b>(js_lexer.TClass)
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-5166" class="path-duplicate">js_parser.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-5166">#L5166</a>: 				p.lexer.<b>Expected</b>(js_lexer.TClass)
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-5382" class="path-duplicate">js_parser.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-5382">#L5382</a>: 				p.lexer.<b>Expected</b>(js_lexer.TClass)
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-5743" class="path-duplicate">js_parser.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-5743">#L5743</a>: 			p.lexer.<b>Expected</b>(js_lexer.TIdentifier)
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-6053" class="path-duplicate">js_parser.go</a><a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/js_parser.go.html#line-6053">#L6053</a>: 							p.lexer.<b>Expected</b>(js_lexer.TClass)
		<a href="../../../../../src/github.com/evanw/esbuild/internal/js_parser/ts_parser.go.html#line-246">ts_parser.go#L246</a>: 					p.lexer.<b>Expected</b>(js_lexer.TIdentifier)
</code></pre><pre id="footer">
<table><tr><td><img src="../../../../../png/go101-twitter.png"></td>
<td>The pages are generated with <a href="https://go101.org/article/tool-golds.html"><b>Golds</b></a> <i>v0.3.2-preview</i>. (GOOS=darwin GOARCH=amd64)
<b>Golds</b> is a <a href="https://go101.org">Go 101</a> project developed by <a href="https://tapirgames.com">Tapir Liu</a>.
PR and bug reports are welcome and can be submitted to <a href="https://github.com/go101/golds">the issue list</a>.
Please follow <a href="https://twitter.com/go100and1">@Go100and1</a> (reachable from the left QR code) to get the latest news of <b>Golds</b>.</td></tr></table></pre>